.github/ISSUE_TEMPLATE/support_request.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Question or Support Request
+about: Something you need help with or isn't clear
+title: ''
+assignees: ''
+
+---
+
+We're happy to help you! However we use Issues only for engineering work to be done.
+
+For questions please use one our support channels,
+- https://github.com/Agoric/agoric-sdk/discussions
+- https://discord.com/invite/agoric-585576150827532298
+
```

Comment:
Our discord forum is another relevant option:

```suggestion
 - https://community.agoric.com/

```

Ideally we'd give a bit of description to help folks choose between these options, but we can start without it.